etc/pam.d/Makefile.am

```diff
@@ -2,20 +2,20 @@
 # and also cooperate to make a distribution for `make dist'
 
 pamd_files = \
+	chage \
 	chfn \
 	chsh \
 	groupmems \
 	login \
+	newusers \
 	passwd
 
 pamd_acct_tools_files = \
-	chage \
```

Comment:
As noted in https://github.com/shadow-maint/shadow/pull/928#issuecomment-1916378713 I believe this should not be changed, but instead `chpasswd`.

Reply:
Corrected chage to chpasswd